cmd/gardenadm/main.go

```diff
@@ -0,0 +1,18 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	"github.com/gardener/gardener/cmd/gardenadm/app"
+	"github.com/gardener/gardener/cmd/utils"
+)
+
+func main() {
+	utils.DeduplicateWarnings()
+
+	_ = app.NewCommand().ExecuteContext(signals.SetupSignalHandler())
```

Comment:
Yes, this is to not duplicate the error message in case the CL is used wrongly. With `gardenadm`, we are printing to `os.Stdout`, but for the other binaries, we use loggers.